kalitectl.py

```diff
@@ -76,6 +76,13 @@
 import re
 import subprocess
 
+# We do not understand --option value, only --option=value.
+# Match all patterns of "--option value" and fail if they exist
+__validate_cmd_options = re.compile(r"--[^\s-]+\s+[^-\s][^-\s]+")
```

Comment:
This is an opaque regex right here. Can we get away with normal substring matching?